Travis test failing

Tests are failing in travis on current master:
```
$ cake test
Running unit tests...
Pass (131/131)
Running DOM tests...
Error: Cannot find module 'util'

  phantomjs://bootstrap.js:289
  phantomjs://bootstrap.js:254 in require
  /home/travis/build/mrmr1993/vimium/node_modules/path/path.js:28
  /home/travis/build/mrmr1993/vimium/node_modules/path/path.js:522
  /home/travis/build/mrmr1993/vimium/node_modules/path/path.js:523
TypeError: 'undefined' is not a function (evaluating 'path.dirname(system.args[0])')

  ./tests/dom_tests/phantom_runner.js:35
  ./tests/dom_tests/phantom_runner.js:60
```

Compare [here](https://travis-ci.org/mrmr1993/vimium/builds/46269081) (a copy of master on my fork) to [here](https://travis-ci.org/philc/vimium/builds/46035786) when the last commit was pushed yesterday.

--- a/.travis.yml
+++ b/.travis.yml
@@ -3,6 +3,7 @@ node_js: 0.10
 before_install:
   - "npm install -g coffee-script"
   - "npm install path"
+  - "npm install util"
   - "cake build"
 script: "cake test"
 notifications:

